modules/phase_field/src/materials/CrossTermBarrierFunctionMaterial.C

```diff
@@ -30,17 +30,20 @@ CrossTermBarrierFunctionMaterial::CrossTermBarrierFunctionMaterial(const InputPa
     _prop_d2g(_num_eta)
 
 {
+  //loop counters
+  unsigned int i,j;
+
```

Comment:
This is probably left over from rebasing, you need to remove these loop counters which are declared outside the loop.